app/shared/components/Button/component.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import classNames from 'classnames'
+
+export default class Button extends React.Component {
+  onClick (event) {
+    const { clickHandler } = this.props
+
+    if (!clickHandler) {
+      return
+    }
+
+    event.preventDefault()
+    event.stopPropagation()
+
+    const handler = clickHandler()
+  }
+
+  render () {
+    let component
+    let data = {}
+    let { children, url, label, type, modifiers } = this.props
+    let classes = classNames('btn', this.props.className, modifiers, {
+      'loading': this.props.disabled && this.props.loading
+    })
+
+    // list out data- and aria- attributes and stick em in
+    for (const prop in this.props) {
```

Comment:
Out of interest - Is this a fix for certain attributes React doesn't support?

Reply:
It didn't like the aria- roles for some reason and kept rejecting it. The data attributes aren't really needed any more as we are not using Bootstrap javascript or jQuery so I should whip that out

Reply:
Oh also I would add that it is a plain old button - so this is the only way (I could think of) to get aria attributes only when needed

Reply:
ok makes sense